# holds multiple pixel paths
std_msgs/Header  				header
vbme_pkg/Pixel2DFlow_msg[] 		Points 			# list of track points. Points.size() is the number of points
